code/game/verbs.dm

```diff
@@ -156,7 +157,8 @@
 	if(!islist(new_verbs))
 		new_verbs = list(new_verbs)
 	var/list/added_verbs = list()
-	for(var/procpath/P as() in new_verbs)
+	for(var/pp in new_verbs)
+		var/procpath/P = pp
```

Comment:
I don't see how this would fix the runtime. Shouldn't you just make it a typed loop?

Reply:
ok you're right, for some reason add_verbs is getting called with null